ark/utils/segmentation_utils_test.py

```diff
@@ -137,3 +161,32 @@ def test_transform_expression_matrix_multiple_compartments():
     for cell in cell_data.cell_id:
         arcsinh_vals = np.arcsinh(cell_data.loc[:, cell, modified_cols].values)
         assert np.array_equal(arcsinh_data.loc[:, cell, modified_cols].values, arcsinh_vals)
+
+
+def test_visualize_watershed():
+    with tempfile.TemporaryDirectory() as temp_dir:
+        channel_xr = _generate_channel_xr()
+        overlay_channels = channel_xr.channels.values[:2],
+        segmentation_labels_xr = _generate_deepcell_ouput()
```

Comment:
I think you've merged in changes from master, which means these can be replaced with 
```
from ark.utils import test_utils
...
channel_xr = test_utils.make_images_xarray(np.zeros((2, 50, 50, 5)))
...
segmentation_labels_xr = test_utils.make_labels_xarray(np.zeros((2, 50, 50, 1)))
```